### PR TITLE
Update release-notes-8.6.asciidoc

### PR DESCRIPTION
**Pull request by nimarezainia**
modifying the release notes for a known issue https://github.com/elastic/elastic-agent/issues/2103

### CODE REVIEW

**Comment by dedemorton** [comment]
@nimarezainia So I've added the backport label for 8.5, but there will be merge conflicts when that PR gets opened. I can take care of getting the change into all the 8.5.x entries.

**Review by dedemorton — COMMENTED** [review]
Just one question....

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 1]
```diff
@@ -174,6 +174,18 @@ For more information, refer to {agent-issue}2066[#2066].
...
+[[known-issue-issue-2103]]
+.Installation of {agent} on MacOS Ventura may fail if Full Disk Access has not been granted to the installer
+[%collapsible]
+====
+*Details* +
+This issue occurs on MacOS Ventura when Full Disk Access is not granted to the application that runs the installation command.
+This could be either a Terminal or any custom package that a user has built to distribute {agent}.
+
+*Impact* +
+{agent} will fail to install and produce "Error: failed to fix permissions: chown elastic-agent.app: operation not permitted" message.
```
[the remark this answers is not in the document]
Comment (nimarezainia) [reply docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:186]: @dedemorton yes there is a workaround. I added the workaround also.

File: docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc [hunk 2]
```diff
@@ -174,6 +174,19 @@ For more information, refer to {agent-issue}2066[#2066].
...
+Ensure that the application (eg. Terminal, or a custom package) used to install {agent} has Full Disk Access before running: sudo ./elastic-agent install
```
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc:187]:
```suggestion
Ensure that the application used to install {agent} (for example, the Terminal or custom package) has Full Disk Access before running `sudo ./elastic-agent install`.
```

**Review by dedemorton — COMMENTED** [review]
I'd change the wording a bit to avoid separating the noun "application" from its modifier.

**Review by dedemorton — APPROVED** [review]
LGTM!

**Pull request merged**